src/node_watchdog.cc

```diff
@@ -13,6 +13,10 @@ Watchdog::Watchdog(v8::Isolate* isolate, uint64_t ms) : isolate_(isolate),
   loop_ = new uv_loop_t;
   CHECK(loop_);
   rc = uv_loop_init(loop_);
+  if (rc != 0) {
+       FatalError("node::Watchdog::Watchdog()",
+               "Failed to initialize loop. Maybe the file limit is reached?");
```

Comment:
I'm not sure about the second part of this error message. If we can't say with certainty what the problem is, I don't think we should leave questions like this.